### PR TITLE
Fixes #21238: Rename "classic nodes" to "unix node"

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/21238
image
image

### CODE REVIEW

**Comment by amousset** [comment]
I think this should be validated by a thread in *produit*. The semantics should be agreed on and shared throughout the team.

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Review by fanf — APPROVED** [review]
OK to me, great enhancement. Update of LDAP works as intended

**Comment by ElaadF** [comment]
PR rebased

File: webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/domain/AgentTypes.scala [hunk 1]
```diff
@@ -161,10 +161,10 @@ object AgentType {
   final case object Dsc extends AgentType {
     override def id           = "dsc"
     override def oldShortName = "dsc"
-    override def displayName  = "Rudder (Windows DSC)"
+    override def displayName  = "Rudder Windows"
     override def toRulesPath  = "/dsc"
     override def inventoryAgentNames   = Set("dsc")
-    override val inventorySoftwareName = "Rudder agent (DSC)"
+    override val inventorySoftwareName = "Rudder agent (Windows)"
```
Comment (ncharles) [inline webapp/sources/ldap-inventory/inventory-api/src/main/scala/com/normation/inventory/domain/AgentTypes.scala:167]: this break detection of windows version

**Pull request merged**